getting-started/securing-timescale-cloud.md

```diff
@@ -0,0 +1,68 @@
+# How to secure network access to your Timescale Cloud service
+
+It's the easiest question to ask, but the hardest to answer about any system: 
+**is it secure?** Having a secure system doesn't have a binary "yes or no" answer 
+like the question implies. Security is implemented in layers and each choice 
+comes with tradeoffs. And in a world where you are using cloud services with your 
+system, this topic becomes even more important and complex.
```

Comment:
The tone here seems not-quite-docs-like.  Maybe better to start with the next paragraph, but change the opening to something like
> One very critical piece of securing your database within Timescale Cloud is network protection.

etc etc